client/app/views/app_layout.coffee

```diff
@@ -14,9 +14,9 @@ require 'normalize.css/normalize.css'
 
 module.exports = class AppLayout extends LayoutView
 
-    template: require './templates/layout_app'
+    template: -> '<main/>'
```

Comment:
Whoo ok, I understand perfectly now. I though it was some kind of magic trick to refer to a 'main' component elsewhere. Let's say it's Friday Fatigue, ok ?